packages/transaction-controller/src/hooks/ExtraTransactionsPublishHook.ts

```diff
@@ -107,7 +107,6 @@ export class ExtraTransactionsPublishHook {
       from,
       networkClientId,
       transactions,
-      useHook: true,
```

Comment:
Even with the new `methods` property, we probably wouldn't want to hardcode it here, maybe we need an optional `batchTransactionMethods` property in `TransactionMeta`?

Though I'd recommend we do that in future, and just hardcode here for now.